app/routes/signup.tsx

```diff
@@ -2,78 +2,40 @@ import { redirect } from "@remix-run/node";
-            placeholder="Enter your email"
-          />
-        </div>
-
-        <div className="mb-4">
-          <label htmlFor="username" className="block text-sm font-medium text-gray-700 dark:text-gray-300">Username</label>
-          <input
-            type="text"
-            id="username"
-            name="username"
-            className="mt-1 block w-full px-3 py-2 border border-gray-300 dark:border-gray-700 rounded-md shadow-sm focus:outline-none focus:ring-blue-500 focus:border-blue-500 sm:text-sm"
-            placeholder="Create a username"
-          />
-        </div>
-
-        <div className="mb-4">
-          <label htmlFor="password" className="block text-sm font-medium text-gray-700 dark:text-gray-300">Password</label>
-          <input
-            type="password"
-            id="password"
-            name="password"
-            className="mt-1 block w-full px-3 py-2 border border-gray-300 dark:border-gray-700 rounded-md shadow-sm focus:outline-none focus:ring-blue-500 focus:border-blue-500 sm:text-sm"
-            placeholder="Create a password"
-          />
-        </div>
-
-        <div className="mb-6">
-          <label htmlFor="confirm-password" className="block text-sm font-medium text-gray-700 dark:text-gray-300">Confirm Password</label>
-          <input
-            type="password"
-            id="confirm-password"
-            name="confirm-password"
-            className="mt-1 block w-full px-3 py-2 border border-gray-300 dark:border-gray-700 rounded-md shadow-sm focus:outline-none focus:ring-blue-500 focus:border-blue-500 sm:text-sm"
-            placeholder="Confirm your password"
-          />
-        </div>
-
-        <button
-          type="submit"
-          className="w-full bg-gradient-to-r from-blue-500 to-purple-600 text-white py-2 px-4 rounded-md hover:from-blue-600 hover:to-purple-700"
-        >
-          Sign Up
-        </button>
-      </form>
+    <div className="sign-up-alt">
+      {/* Header with logo and title */}
+      <header className="header">
+        <img className="logo" src="app/assets/logo/sobremesa.svg" alt="Sobremesa Logo" />
+        <h2 className="header-title">Sobremesa</h2>
+      </header>
+
+      {/* Form inside a rounded box */}
+      <div className="form-box">
+        <h1 className="title">Create your account</h1>
+        <form className="form">
+          <input className="input" placeholder="Username" type="text" />
+          <input className="input" placeholder="Full Name" type="text" />
+          <input className="input" placeholder="Email" type="email" />
+          <input className="input" placeholder="Password" type="password" />
+          <div className="terms">
+            <input type="checkbox" id="terms" />
+            <label htmlFor="terms">
+              Yes I agree to the <a href="#">Terms & Conditions</a>
```

Comment:
Should create a redirect for "Terms and Conditions". It can lead to a blank page that we will will with terms and conditions later or simply create a pop-up that will contain our terms and conditions.